src/code_f2470_len_27f0.c

```diff
@@ -100,9 +100,45 @@ ApiStatus SetNpcJumpscale(ScriptInstance* script, s32 isInitialCall) {
     return ApiStatus_DONE2;
 }
 
-INCLUDE_API_ASM(code_f2470_len_27f0, SetNpcAnimation);
+ApiStatus SetNpcAnimation(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* ptrReadPos = script->ptrReadPos;
+    NpcId npcID = get_variable(script, *ptrReadPos++);
+    s32 animation = get_variable(script, *ptrReadPos);
+    Npc* npcPtr = resolve_npc(script, npcID);
+
+    if (npcPtr != NULL) {
+        set_npc_animation(npcPtr, animation);
+        return ApiStatus_DONE2; // Doesn't match if omitted
+    }
+    return ApiStatus_DONE2;
+}
+
+ApiStatus GetNpcAnimation(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* ptrReadPos = script->ptrReadPos;
+    NpcId npcID = get_variable(script, *ptrReadPos++);
+    Bytecode outVar = *ptrReadPos++;
+    Npc* npcPtr = resolve_npc(script, npcID);
+
+    if (npcPtr != NULL) {
+        set_variable(script, outVar, npcPtr->currentAnim);
+        return ApiStatus_DONE2; // Doesn't match if omitted
+    }
+    return ApiStatus_DONE2;
+}
+
+// TODO: Figure out meaning of float value in Npc struct that this function is setting
+ApiStatus func_802CE1C0(ScriptInstance* script, s32 isInitialCall) {
```

Comment:
Note: func_802CE1C0 was in GetNpcAnimation.s, which is why there was not a file removed for it.

Reply:
This function is only ever used twice ingame - and it's in the Lava Piranha fight on the Kolerado NPC:
![image](https://user-images.githubusercontent.com/9429556/90570339-60eaa580-e1a7-11ea-9958-5a67473fd856.png)
I imagine it has something to do with the NPC's scale factor...

Reply:
[Link to part of the game where this occurs](https://youtu.be/pdrhCW7qAZc?t=42505)

Reply:
Added commits (below) to change the unknown float value to animationSpeed